doc/modules/language-guide/pages/motoko-introduction.adoc

```diff
@@ -14,7 +14,7 @@ Although aimed squarely at the {IC}, its design is general enough to support fut
 == Asynchronous messaging and type sound execution
 
 {proglang} permits modern programming idioms, including special programming abstractions for distributed applications (dapps).
-Each dapp consists of one or more  _actor_s that communicate solely by asynchronous message passing. The state of an actor is isolated from all other actors, supporting distribution. There is no way to share state between several actors. 
+Each dapp consists of one or more  _actors_ that communicate solely by asynchronous message passing. The state of an actor is isolated from all other actors, supporting distribution. There is no way to share state between several actors. 
```

Comment:
@crusso I think the intention here was to set `actor` as teletype and the "s" (for plural) as regular. @christyjacob4 can you drop a small .PNG how this revered before and after?

Reply:
Absolutely 👍

Reply:
Before ( in page https://smartcontracts.org/docs/language-guide/motoko-introduction.html#_asynchronous_messaging_and_type_sound_execution )

<img width="891" alt="Screenshot 2021-12-07 at 3 57 04 AM" src="https://user-images.githubusercontent.com/20852629/144941471-e060d4c6-3536-49ba-8f0a-862a860e8d6f.png">
 

After ( in page https://github.com/christyjacob4/motoko/blob/patch-2/doc/modules/language-guide/pages/motoko-introduction.adoc#asynchronous-messaging-and-type-sound-execution )
<img width="935" alt="Screenshot 2021-12-07 at 3 59 30 AM" src="https://user-images.githubusercontent.com/20852629/144941663-92721195-edd8-411d-a35f-5d0f846b0822.png">


Unfortunately I could not find the contribution guidelines to generate the docs locally. Would it make sense for me to add a `Contributing.md` to this repo ? If yes, where can we have a discussion on what topics need to be covered in the guidelines (apart from the obvious how to setup / build / run docs locally ) ?

Reply:
I think you are doing the right thing. Just file a pull request.

Reply:
I found a reference that could help: https://docs.asciidoctor.org/asciidoc/latest/text/quotation-marks-and-apostrophes/#possessive-monospace
Possibly one has towrite double backticks around `actor` like this:
``` adoc
  ... ``actor``s ...
```

Reply:
If you want to see how this is rendered, you can select "View file" from the menu right to the "Viewed" checkbox above.

Reply:
going by the underscore character that was already present before my chance. I assume the intention was to have actors in italics _actors_ . 

Correct me if I'm wrong.

Reply:
No, that right. Sorry to be so late to the party.

Reply:
```suggestion
Each dapp consists of one or more  _actors_ that communicate solely by _asynchronous message passing_. The state of an actor is isolated from all other actors, supporting distribution. There is no way to share state between several actors. 
```